### PR TITLE
Fix typo in tx notification, add notification route

### PR DESCRIPTION
**Pull request by mradkov**

### CODE REVIEW

File: src/popup/router/pages/Notifications.vue [hunk 1]
```diff
@@ -3,7 +3,9 @@
     <ae-list class="noti-list">
       <ae-list-item fill="neutral" v-for="(noti, i) in notifications" :key="i" class="noti">
         <img src="../../../icons/icon_48.png" />
-        <span> {{ noti.content }} </span>
+        <span>
+          <router-link :to="noti.route"> {{ noti.content }} </router-link>
+        </span>
```
Comment (davidyuk) [inline src/popup/router/pages/Notifications.vue]: I propose to don't use `router-link` if `noti.route` is empty:
```suggestion
        <Component
          :is="noti.route ? 'RouterLink' : 'span'"
          :to="noti.route"
        >
          {{ noti.content }}
        </span>
```

File: src/popup/locales/cn.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
             "activity": "活动",
             "dashboard": "仪表板",
             "aeid": "❤币",
-            "coronaWallet": "SuperHero钱包",
+            "superheroWallet": "SuperHero钱包",
```
Comment (davidyuk) [inline src/popup/locales/cn.json]: Would be more flexible to rename it to something like `walletName` 😉

Comment (mradkov) [reply src/popup/locales/cn.json]: Sure, I agree.

File: src/popup/router/pages/Notifications.vue (hunk 1 above)
Comment (mradkov) [reply src/popup/router/pages/Notifications.vue]: This is breaking the build though.

Comment (mradkov) [reply src/popup/router/pages/Notifications.vue]: Should be ok now after https://github.com/aeternity/superhero-wallet/pull/77/commits/2430ccc04abf3dcd5855c4f83ba77751be35d309

File: src/popup/router/pages/Notifications.vue [hunk 3]
```diff
@@ -3,7 +3,11 @@
     <ae-list class="noti-list">
       <ae-list-item fill="neutral" v-for="(noti, i) in notifications" :key="i" class="noti">
         <img src="../../../icons/icon_48.png" />
-        <span> {{ noti.content }} </span>
+        <span>
+          <Component :is="noti.route ? 'RouterLink' : 'span'" :to="noti.route">
+            {{ noti.content }}
+          </Component>
+        </span>
```
Comment (davidyuk) [inline src/popup/router/pages/Notifications.vue]:
```suggestion
        <Component :is="noti.route ? 'RouterLink' : 'span'" :to="noti.route">
          {{ noti.content }}
        </Component>
```
Aha, I forgot to close Component tag, can we try this?

**Pull request merged**